2016 Primary Election Merced test failure

Test fails on U.S. House results for Merced

```
E                   AssertionError: U.S. House for Merced failed
E                   assert {'David Roger...herra': 13917} == {'David Rogers...herra': 13920}
E                     Omitting 2 identical items, use -v to show
E                     Differing items:
E                     {'Johnny M. Tacherra': 13917} != {'Johnny M. Tacherra': 13920}

```

--- a/2016/counties/20160607__ca__primary__merced__precinct.csv
+++ b/2016/counties/20160607__ca__primary__merced__precinct.csv
@@ -724,6 +724,7 @@ Merced,2019,President,,REP,Ted Cruz,7
 Merced,2019,Proposition 50,,,No,59
 Merced,2019,Proposition 50,,,Yes,152
 Merced,2019,State Assembly,21,DEM,Adam Gray,123
+Merced,2019,State Assembly,21,,Brien J. Rahilly,3
 Merced,2019,State Assembly,21,REP,Greg Opinski,80
 Merced,2019,U.S. House,16,REP,David Rogers,35
 Merced,2019,U.S. House,16,DEM,Jim Costa,78
@@ -762,6 +763,7 @@ Merced,2020,President,,REP,Ted Cruz,11
 Merced,2020,Proposition 50,,,No,46
 Merced,2020,Proposition 50,,,Yes,171
 Merced,2020,State Assembly,21,DEM,Adam Gray,125
+Merced,2020,State Assembly,21,,Brien J. Rahilly,1
 Merced,2020,State Assembly,21,REP,Greg Opinski,87
 Merced,2020,U.S. House,16,REP,David Rogers,30
 Merced,2020,U.S. House,16,DEM,Jim Costa,77
@@ -957,6 +959,7 @@ Merced,2025,President,,DEM,Willie Wilson,3
 Merced,2025,Proposition 50,,,No,89
 Merced,2025,Proposition 50,,,Yes,354
 Merced,2025,State Assembly,21,DEM,Adam Gray,303
+Merced,2025,State Assembly,21,,Brien J. Rahilly,2
 Merced,2025,State Assembly,21,REP,Greg Opinski,151
 Merced,2025,U.S. House,16,REP,David Rogers,69
 Merced,2025,U.S. House,16,DEM,Jim Costa,232
@@ -1007,6 +1010,7 @@ Merced,2026,President,,REP,Ted Cruz,3
 Merced,2026,Proposition 50,,,No,31
 Merced,2026,Proposition 50,,,Yes,183
 Merced,2026,State Assembly,21,DEM,Adam Gray,145
+Merced,2026,State Assembly,21,,Brien J. Rahilly,1
 Merced,2026,State Assembly,21,REP,Greg Opinski,74
 Merced,2026,U.S. House,16,REP,David Rogers,32
 Merced,2026,U.S. House,16,DEM,Jim Costa,116
@@ -1054,6 +1058,7 @@ Merced,2027,President,,DEM,Willie Wilson,2
 Merced,2027,Proposition 50,,,No,76
 Merced,2027,Proposition 50,,,Yes,290
 Merced,2027,State Assembly,21,DEM,Adam Gray,230
+Merced,2027,State Assembly,21,,Brien J. Rahilly,3
 Merced,2027,State Assembly,21,REP,Greg Opinski,142
 Merced,2027,U.S. House,16,REP,David Rogers,62
 Merced,2027,U.S. House,16,DEM,Jim Costa,185
@@ -1233,6 +1238,7 @@ Merced,2031,President,,DEM,Willie Wilson,1
 Merced,2031,Proposition 50,,,No,61
 Merced,2031,Proposition 50,,,Yes,263
 Merced,2031,State Assembly,21,DEM,Adam Gray,215
+Merced,2031,State Assembly,21,,Brien J. Rahilly,2
 Merced,2031,State Assembly,21,REP,Greg Opinski,107
 Merced,2031,U.S. House,16,REP,David Rogers,43
 Merced,2031,U.S. House,16,DEM,Jim Costa,183
@@ -1502,6 +1508,7 @@ Merced,2037,President,,DEM,Willie Wilson,1
 Merced,2037,Proposition 50,,,No,45
 Merced,2037,Proposition 50,,,Yes,286
 Merced,2037,State Assembly,21,DEM,Adam Gray,219
+Merced,2037,State Assembly,21,,Brien J. Rahilly,1
 Merced,2037,State Assembly,21,REP,Greg Opinski,119
 Merced,2037,U.S. House,16,REP,David Rogers,51
 Merced,2037,U.S. House,16,DEM,Jim Costa,156
@@ -1676,6 +1683,7 @@ Merced,2041,State Assembly,21,REP,Greg Opinski,135
 Merced,2041,U.S. House,16,REP,David Rogers,78
 Merced,2041,U.S. House,16,DEM,Jim Costa,200
 Merced,2041,U.S. House,16,REP,Johnny M. Tacherra,127
+Merced,2041,U.S. House,16,,Richard Gomez,1
 Merced,2041,U.S. Senate,,IND,Clive Grey,5
 Merced,2041,U.S. Senate,,REP,Don Krampe,5
 Merced,2041,U.S. Senate,,REP,Duf Sundheim,36
@@ -1776,6 +1784,7 @@ Merced,2043,President,,DEM,Willie Wilson,1
 Merced,2043,Proposition 50,,,No,90
 Merced,2043,Proposition 50,,,Yes,279
 Merced,2043,State Assembly,21,DEM,Adam Gray,223
+Merced,2043,State Assembly,21,,Brien J. Rahilly,1
 Merced,2043,State Assembly,21,REP,Greg Opinski,151
 Merced,2043,U.S. House,16,REP,David Rogers,76
 Merced,2043,U.S. House,16,DEM,Jim Costa,165
@@ -1999,6 +2008,7 @@ Merced,2048,President,,AIP,Wiley Drake,1
 Merced,2048,Proposition 50,,,No,41
 Merced,2048,Proposition 50,,,Yes,214
 Merced,2048,State Assembly,21,DEM,Adam Gray,161
+Merced,2048,State Assembly,21,,Brien J. Rahilly,1
 Merced,2048,State Assembly,21,REP,Greg Opinski,101
 Merced,2048,U.S. House,16,REP,David Rogers,49
 Merced,2048,U.S. House,16,DEM,Jim Costa,102
@@ -2042,6 +2052,7 @@ Merced,2049,President,,DEM,Willie Wilson,1
 Merced,2049,Proposition 50,,,No,51
 Merced,2049,Proposition 50,,,Yes,242
 Merced,2049,State Assembly,21,DEM,Adam Gray,213
+Merced,2049,State Assembly,21,,Brien J. Rahilly,1
 Merced,2049,State Assembly,21,REP,Greg Opinski,97
 Merced,2049,U.S. House,16,REP,David Rogers,42
 Merced,2049,U.S. House,16,DEM,Jim Costa,157
@@ -2086,6 +2097,7 @@ Merced,2050,President,,DEM,Willie Wilson,3
 Merced,2050,Proposition 50,,,No,53
 Merced,2050,Proposition 50,,,Yes,201
 Merced,2050,State Assembly,21,DEM,Adam Gray,176
+Merced,2050,State Assembly,21,,Brien J. Rahilly,4
 Merced,2050,State Assembly,21,REP,Greg Opinski,95
 Merced,2050,U.S. House,16,REP,David Rogers,41
 Merced,2050,U.S. House,16,DEM,Jim Costa,158
@@ -2275,6 +2287,7 @@ Merced,3055,President,,DEM,Willie Wilson,1
 Merced,3055,Proposition 50,,,No,63
 Merced,3055,Proposition 50,,,Yes,265
 Merced,3055,State Assembly,21,DEM,Adam Gray,216
+Merced,3055,State Assembly,21,,Brien J. Rahilly,1
 Merced,3055,State Assembly,21,REP,Greg Opinski,116
 Merced,3055,U.S. House,16,REP,David Rogers,48
 Merced,3055,U.S. House,16,DEM,Jim Costa,187
@@ -2876,6 +2889,7 @@ Merced,3068,President,,AIP,Wiley Drake,1
 Merced,3068,Proposition 50,,,No,25
 Merced,3068,Proposition 50,,,Yes,92
 Merced,3068,State Assembly,21,DEM,Adam Gray,62
+Merced,3068,State Assembly,21,,Brien J. Rahilly,1
 Merced,3068,State Assembly,21,REP,Greg Opinski,57
 Merced,3068,U.S. House,16,REP,David Rogers,12
 Merced,3068,U.S. House,16,DEM,Jim Costa,49
@@ -2913,6 +2927,7 @@ Merced,3069,President,,DEM,Willie Wilson,1
 Merced,3069,Proposition 50,,,No,67
 Merced,3069,Proposition 50,,,Yes,251
 Merced,3069,State Assembly,21,DEM,Adam Gray,238
+Merced,3069,State Assembly,21,,Brien J. Rahilly,2
 Merced,3069,State Assembly,21,REP,Greg Opinski,98
 Merced,3069,U.S. House,16,REP,David Rogers,38
 Merced,3069,U.S. House,16,DEM,Jim Costa,228
@@ -3016,6 +3031,7 @@ Merced,3071,President,,REP,Ted Cruz,16
 Merced,3071,Proposition 50,,,No,59
 Merced,3071,Proposition 50,,,Yes,311
 Merced,3071,State Assembly,21,DEM,Adam Gray,206
+Merced,3071,State Assembly,21,,Brien J. Rahilly,1
 Merced,3071,State Assembly,21,REP,Greg Opinski,179
 Merced,3071,U.S. House,16,REP,David Rogers,50
 Merced,3071,U.S. House,16,DEM,Jim Costa,139
@@ -3497,6 +3513,7 @@ Merced,4082,President,,AIP,Wiley Drake,1
 Merced,4082,Proposition 50,,,No,89
 Merced,4082,Proposition 50,,,Yes,300
 Merced,4082,State Assembly,21,DEM,Adam Gray,313
+Merced,4082,State Assembly,21,,Brien J. Rahilly,2
 Merced,4082,State Assembly,21,REP,Greg Opinski,91
 Merced,4082,U.S. House,16,REP,David Rogers,41
 Merced,4082,U.S. House,16,DEM,Jim Costa,314
@@ -4667,7 +4684,7 @@ Merced,5108,State Assembly,21,DEM,Adam Gray,222
 Merced,5108,State Assembly,21,REP,Greg Opinski,107
 Merced,5108,U.S. House,16,REP,David Rogers,19
 Merced,5108,U.S. House,16,DEM,Jim Costa,197
-Merced,5108,U.S. House,16,REP,Johnny M. Tacherra,133
+Merced,5108,U.S. House,16,REP,Johnny M. Tacherra,130
 Merced,5108,U.S. Senate,,IND,Clive Grey,1
 Merced,5108,U.S. Senate,,REP,Don Krampe,5
 Merced,5108,U.S. Senate,,REP,Duf Sundheim,16
@@ -5359,6 +5376,7 @@ Merced,91131,President,,REP,Ted Cruz,1
 Merced,91131,Proposition 50,,,No,2
 Merced,91131,Proposition 50,,,Yes,15
 Merced,91131,State Assembly,21,DEM,Adam Gray,2
+Merced,91131,State Assembly,21,,Brien J. Rahilly,1
 Merced,91131,State Assembly,21,REP,Greg Opinski,14
 Merced,91131,U.S. House,16,REP,David Rogers,7
 Merced,91131,U.S. House,16,DEM,Jim Costa,1
@@ -5577,6 +5595,7 @@ Merced,91142,President,,REP,Ted Cruz,2
 Merced,91142,Proposition 50,,,No,21
 Merced,91142,Proposition 50,,,Yes,75
 Merced,91142,State Assembly,21,DEM,Adam Gray,40
+Merced,91142,State Assembly,21,,Brien J. Rahilly,2
 Merced,91142,State Assembly,21,REP,Greg Opinski,58
 Merced,91142,U.S. House,16,REP,David Rogers,34
 Merced,91142,U.S. House,16,DEM,Jim Costa,13
@@ -5869,6 +5888,7 @@ Merced,91162,President,,DEM,Michael Steinberg,1
 Merced,91162,Proposition 50,,,No,3
 Merced,91162,Proposition 50,,,Yes,34
 Merced,91162,State Assembly,21,DEM,Adam Gray,20
+Merced,91162,State Assembly,21,,Brien J. Rahilly,2
 Merced,91162,State Assembly,21,REP,Greg Opinski,19
 Merced,91162,U.S. House,16,REP,David Rogers,17
 Merced,91162,U.S. House,16,DEM,Jim Costa,1
@@ -5994,6 +6014,7 @@ Merced,92169,President,,DEM,Willie Wilson,1
 Merced,92169,Proposition 50,,,No,15
 Merced,92169,Proposition 50,,,Yes,81
 Merced,92169,State Assembly,21,DEM,Adam Gray,52
+Merced,92169,State Assembly,21,,Brien J. Rahilly,1
 Merced,92169,State Assembly,21,REP,Greg Opinski,39
 Merced,92169,U.S. House,16,REP,David Rogers,15
 Merced,92169,U.S. House,16,DEM,Jim Costa,33
@@ -6236,6 +6257,7 @@ Merced,92182,President,,REP,Ted Cruz,2
 Merced,92182,Proposition 50,,,No,7
 Merced,92182,Proposition 50,,,Yes,27
 Merced,92182,State Assembly,21,DEM,Adam Gray,26
+Merced,92182,State Assembly,21,,Brien J. Rahilly,1
 Merced,92182,State Assembly,21,REP,Greg Opinski,8
 Merced,92182,U.S. House,16,REP,David Rogers,5
 Merced,92182,U.S. House,16,DEM,Jim Costa,28

--- a/2016/counties/20160607__ca__primary__merced__precinct.csv
+++ b/2016/counties/20160607__ca__primary__merced__precinct.csv
@@ -454,6 +454,7 @@ Merced,1012,President,,DEM,Willie Wilson,1
 Merced,1012,Proposition 50,,,No,22
 Merced,1012,Proposition 50,,,Yes,95
 Merced,1012,State Assembly,21,DEM,Adam Gray,80
+Merced,1012,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,1012,State Assembly,21,REP,Greg Opinski,31
 Merced,1012,U.S. House,16,REP,David Rogers,16
 Merced,1012,U.S. House,16,DEM,Jim Costa,67
@@ -724,7 +725,7 @@ Merced,2019,President,,REP,Ted Cruz,7
 Merced,2019,Proposition 50,,,No,59
 Merced,2019,Proposition 50,,,Yes,152
 Merced,2019,State Assembly,21,DEM,Adam Gray,123
-Merced,2019,State Assembly,21,,Brien J. Rahilly,3
+Merced,2019,State Assembly,21,REP,Brien J. Rahilly,3
 Merced,2019,State Assembly,21,REP,Greg Opinski,80
 Merced,2019,U.S. House,16,REP,David Rogers,35
 Merced,2019,U.S. House,16,DEM,Jim Costa,78
@@ -763,7 +764,7 @@ Merced,2020,President,,REP,Ted Cruz,11
 Merced,2020,Proposition 50,,,No,46
 Merced,2020,Proposition 50,,,Yes,171
 Merced,2020,State Assembly,21,DEM,Adam Gray,125
-Merced,2020,State Assembly,21,,Brien J. Rahilly,1
+Merced,2020,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,2020,State Assembly,21,REP,Greg Opinski,87
 Merced,2020,U.S. House,16,REP,David Rogers,30
 Merced,2020,U.S. House,16,DEM,Jim Costa,77
@@ -959,7 +960,7 @@ Merced,2025,President,,DEM,Willie Wilson,3
 Merced,2025,Proposition 50,,,No,89
 Merced,2025,Proposition 50,,,Yes,354
 Merced,2025,State Assembly,21,DEM,Adam Gray,303
-Merced,2025,State Assembly,21,,Brien J. Rahilly,2
+Merced,2025,State Assembly,21,REP,Brien J. Rahilly,2
 Merced,2025,State Assembly,21,REP,Greg Opinski,151
 Merced,2025,U.S. House,16,REP,David Rogers,69
 Merced,2025,U.S. House,16,DEM,Jim Costa,232
@@ -1010,7 +1011,7 @@ Merced,2026,President,,REP,Ted Cruz,3
 Merced,2026,Proposition 50,,,No,31
 Merced,2026,Proposition 50,,,Yes,183
 Merced,2026,State Assembly,21,DEM,Adam Gray,145
-Merced,2026,State Assembly,21,,Brien J. Rahilly,1
+Merced,2026,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,2026,State Assembly,21,REP,Greg Opinski,74
 Merced,2026,U.S. House,16,REP,David Rogers,32
 Merced,2026,U.S. House,16,DEM,Jim Costa,116
@@ -1058,7 +1059,7 @@ Merced,2027,President,,DEM,Willie Wilson,2
 Merced,2027,Proposition 50,,,No,76
 Merced,2027,Proposition 50,,,Yes,290
 Merced,2027,State Assembly,21,DEM,Adam Gray,230
-Merced,2027,State Assembly,21,,Brien J. Rahilly,3
+Merced,2027,State Assembly,21,REP,Brien J. Rahilly,3
 Merced,2027,State Assembly,21,REP,Greg Opinski,142
 Merced,2027,U.S. House,16,REP,David Rogers,62
 Merced,2027,U.S. House,16,DEM,Jim Costa,185
@@ -1238,7 +1239,7 @@ Merced,2031,President,,DEM,Willie Wilson,1
 Merced,2031,Proposition 50,,,No,61
 Merced,2031,Proposition 50,,,Yes,263
 Merced,2031,State Assembly,21,DEM,Adam Gray,215
-Merced,2031,State Assembly,21,,Brien J. Rahilly,2
+Merced,2031,State Assembly,21,REP,Brien J. Rahilly,2
 Merced,2031,State Assembly,21,REP,Greg Opinski,107
 Merced,2031,U.S. House,16,REP,David Rogers,43
 Merced,2031,U.S. House,16,DEM,Jim Costa,183
@@ -1508,7 +1509,7 @@ Merced,2037,President,,DEM,Willie Wilson,1
 Merced,2037,Proposition 50,,,No,45
 Merced,2037,Proposition 50,,,Yes,286
 Merced,2037,State Assembly,21,DEM,Adam Gray,219
-Merced,2037,State Assembly,21,,Brien J. Rahilly,1
+Merced,2037,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,2037,State Assembly,21,REP,Greg Opinski,119
 Merced,2037,U.S. House,16,REP,David Rogers,51
 Merced,2037,U.S. House,16,DEM,Jim Costa,156
@@ -1784,7 +1785,7 @@ Merced,2043,President,,DEM,Willie Wilson,1
 Merced,2043,Proposition 50,,,No,90
 Merced,2043,Proposition 50,,,Yes,279
 Merced,2043,State Assembly,21,DEM,Adam Gray,223
-Merced,2043,State Assembly,21,,Brien J. Rahilly,1
+Merced,2043,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,2043,State Assembly,21,REP,Greg Opinski,151
 Merced,2043,U.S. House,16,REP,David Rogers,76
 Merced,2043,U.S. House,16,DEM,Jim Costa,165
@@ -2008,7 +2009,7 @@ Merced,2048,President,,AIP,Wiley Drake,1
 Merced,2048,Proposition 50,,,No,41
 Merced,2048,Proposition 50,,,Yes,214
 Merced,2048,State Assembly,21,DEM,Adam Gray,161
-Merced,2048,State Assembly,21,,Brien J. Rahilly,1
+Merced,2048,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,2048,State Assembly,21,REP,Greg Opinski,101
 Merced,2048,U.S. House,16,REP,David Rogers,49
 Merced,2048,U.S. House,16,DEM,Jim Costa,102
@@ -2052,7 +2053,7 @@ Merced,2049,President,,DEM,Willie Wilson,1
 Merced,2049,Proposition 50,,,No,51
 Merced,2049,Proposition 50,,,Yes,242
 Merced,2049,State Assembly,21,DEM,Adam Gray,213
-Merced,2049,State Assembly,21,,Brien J. Rahilly,1
+Merced,2049,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,2049,State Assembly,21,REP,Greg Opinski,97
 Merced,2049,U.S. House,16,REP,David Rogers,42
 Merced,2049,U.S. House,16,DEM,Jim Costa,157
@@ -2097,7 +2098,7 @@ Merced,2050,President,,DEM,Willie Wilson,3
 Merced,2050,Proposition 50,,,No,53
 Merced,2050,Proposition 50,,,Yes,201
 Merced,2050,State Assembly,21,DEM,Adam Gray,176
-Merced,2050,State Assembly,21,,Brien J. Rahilly,4
+Merced,2050,State Assembly,21,REP,Brien J. Rahilly,4
 Merced,2050,State Assembly,21,REP,Greg Opinski,95
 Merced,2050,U.S. House,16,REP,David Rogers,41
 Merced,2050,U.S. House,16,DEM,Jim Costa,158
@@ -2287,7 +2288,7 @@ Merced,3055,President,,DEM,Willie Wilson,1
 Merced,3055,Proposition 50,,,No,63
 Merced,3055,Proposition 50,,,Yes,265
 Merced,3055,State Assembly,21,DEM,Adam Gray,216
-Merced,3055,State Assembly,21,,Brien J. Rahilly,1
+Merced,3055,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,3055,State Assembly,21,REP,Greg Opinski,116
 Merced,3055,U.S. House,16,REP,David Rogers,48
 Merced,3055,U.S. House,16,DEM,Jim Costa,187
@@ -2889,7 +2890,7 @@ Merced,3068,President,,AIP,Wiley Drake,1
 Merced,3068,Proposition 50,,,No,25
 Merced,3068,Proposition 50,,,Yes,92
 Merced,3068,State Assembly,21,DEM,Adam Gray,62
-Merced,3068,State Assembly,21,,Brien J. Rahilly,1
+Merced,3068,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,3068,State Assembly,21,REP,Greg Opinski,57
 Merced,3068,U.S. House,16,REP,David Rogers,12
 Merced,3068,U.S. House,16,DEM,Jim Costa,49
@@ -2927,7 +2928,7 @@ Merced,3069,President,,DEM,Willie Wilson,1
 Merced,3069,Proposition 50,,,No,67
 Merced,3069,Proposition 50,,,Yes,251
 Merced,3069,State Assembly,21,DEM,Adam Gray,238
-Merced,3069,State Assembly,21,,Brien J. Rahilly,2
+Merced,3069,State Assembly,21,REP,Brien J. Rahilly,2
 Merced,3069,State Assembly,21,REP,Greg Opinski,98
 Merced,3069,U.S. House,16,REP,David Rogers,38
 Merced,3069,U.S. House,16,DEM,Jim Costa,228
@@ -3031,7 +3032,7 @@ Merced,3071,President,,REP,Ted Cruz,16
 Merced,3071,Proposition 50,,,No,59
 Merced,3071,Proposition 50,,,Yes,311
 Merced,3071,State Assembly,21,DEM,Adam Gray,206
-Merced,3071,State Assembly,21,,Brien J. Rahilly,1
+Merced,3071,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,3071,State Assembly,21,REP,Greg Opinski,179
 Merced,3071,U.S. House,16,REP,David Rogers,50
 Merced,3071,U.S. House,16,DEM,Jim Costa,139
@@ -3610,7 +3611,7 @@ Merced,4084,President,,DEM,Willie Wilson,1
 Merced,4084,Proposition 50,,,No,73
 Merced,4084,Proposition 50,,,Yes,173
 Merced,4084,State Assembly,21,DEM,Adam Gray,136
-Merced,4084,State Assembly,21,,Brien J. Rahilly,2
+Merced,4084,State Assembly,21,REP,Brien J. Rahilly,2
 Merced,4084,State Assembly,21,REP,Greg Opinski,119
 Merced,4084,U.S. House,16,REP,David Rogers,24
 Merced,4084,U.S. House,16,DEM,Jim Costa,86
@@ -5376,7 +5377,7 @@ Merced,91131,President,,REP,Ted Cruz,1
 Merced,91131,Proposition 50,,,No,2
 Merced,91131,Proposition 50,,,Yes,15
 Merced,91131,State Assembly,21,DEM,Adam Gray,2
-Merced,91131,State Assembly,21,,Brien J. Rahilly,1
+Merced,91131,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,91131,State Assembly,21,REP,Greg Opinski,14
 Merced,91131,U.S. House,16,REP,David Rogers,7
 Merced,91131,U.S. House,16,DEM,Jim Costa,1
@@ -5595,7 +5596,7 @@ Merced,91142,President,,REP,Ted Cruz,2
 Merced,91142,Proposition 50,,,No,21
 Merced,91142,Proposition 50,,,Yes,75
 Merced,91142,State Assembly,21,DEM,Adam Gray,40
-Merced,91142,State Assembly,21,,Brien J. Rahilly,2
+Merced,91142,State Assembly,21,REP,Brien J. Rahilly,2
 Merced,91142,State Assembly,21,REP,Greg Opinski,58
 Merced,91142,U.S. House,16,REP,David Rogers,34
 Merced,91142,U.S. House,16,DEM,Jim Costa,13
@@ -5860,7 +5861,7 @@ Merced,91161,President,,REP,Ted Cruz,6
 Merced,91161,Proposition 50,,,No,25
 Merced,91161,Proposition 50,,,Yes,55
 Merced,91161,State Assembly,21,DEM,Adam Gray,33
-Merced,91161,State Assembly,21,,Brien J. Rahilly,2
+Merced,91161,State Assembly,21,REP,Brien J. Rahilly,2
 Merced,91161,State Assembly,21,REP,Greg Opinski,48
 Merced,91161,U.S. House,16,REP,David Rogers,24
 Merced,91161,U.S. House,16,DEM,Jim Costa,15
@@ -6014,7 +6015,7 @@ Merced,92169,President,,DEM,Willie Wilson,1
 Merced,92169,Proposition 50,,,No,15
 Merced,92169,Proposition 50,,,Yes,81
 Merced,92169,State Assembly,21,DEM,Adam Gray,52
-Merced,92169,State Assembly,21,,Brien J. Rahilly,1
+Merced,92169,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,92169,State Assembly,21,REP,Greg Opinski,39
 Merced,92169,U.S. House,16,REP,David Rogers,15
 Merced,92169,U.S. House,16,DEM,Jim Costa,33
@@ -6444,7 +6445,7 @@ Merced,92189,President,,REP,Ted Cruz,10
 Merced,92189,Proposition 50,,,No,21
 Merced,92189,Proposition 50,,,Yes,111
 Merced,92189,State Assembly,21,DEM,Adam Gray,95
-Merced,92189,State Assembly,21,,Brien J. Rahilly,1
+Merced,92189,State Assembly,21,REP,Brien J. Rahilly,1
 Merced,92189,State Assembly,21,REP,Greg Opinski,34
 Merced,92189,U.S. House,16,REP,David Rogers,18
 Merced,92189,U.S. House,16,DEM,Jim Costa,77

--- a/2016/counties/20160607__ca__primary__merced__precinct.csv
+++ b/2016/counties/20160607__ca__primary__merced__precinct.csv
@@ -3513,7 +3513,6 @@ Merced,4082,President,,AIP,Wiley Drake,1
 Merced,4082,Proposition 50,,,No,89
 Merced,4082,Proposition 50,,,Yes,300
 Merced,4082,State Assembly,21,DEM,Adam Gray,313
-Merced,4082,State Assembly,21,,Brien J. Rahilly,2
 Merced,4082,State Assembly,21,REP,Greg Opinski,91
 Merced,4082,U.S. House,16,REP,David Rogers,41
 Merced,4082,U.S. House,16,DEM,Jim Costa,314
@@ -3611,6 +3610,7 @@ Merced,4084,President,,DEM,Willie Wilson,1
 Merced,4084,Proposition 50,,,No,73
 Merced,4084,Proposition 50,,,Yes,173
 Merced,4084,State Assembly,21,DEM,Adam Gray,136
+Merced,4084,State Assembly,21,,Brien J. Rahilly,2
 Merced,4084,State Assembly,21,REP,Greg Opinski,119
 Merced,4084,U.S. House,16,REP,David Rogers,24
 Merced,4084,U.S. House,16,DEM,Jim Costa,86
@@ -5860,6 +5860,7 @@ Merced,91161,President,,REP,Ted Cruz,6
 Merced,91161,Proposition 50,,,No,25
 Merced,91161,Proposition 50,,,Yes,55
 Merced,91161,State Assembly,21,DEM,Adam Gray,33
+Merced,91161,State Assembly,21,,Brien J. Rahilly,2
 Merced,91161,State Assembly,21,REP,Greg Opinski,48
 Merced,91161,U.S. House,16,REP,David Rogers,24
 Merced,91161,U.S. House,16,DEM,Jim Costa,15
@@ -5888,7 +5889,6 @@ Merced,91162,President,,DEM,Michael Steinberg,1
 Merced,91162,Proposition 50,,,No,3
 Merced,91162,Proposition 50,,,Yes,34
 Merced,91162,State Assembly,21,DEM,Adam Gray,20
-Merced,91162,State Assembly,21,,Brien J. Rahilly,2
 Merced,91162,State Assembly,21,REP,Greg Opinski,19
 Merced,91162,U.S. House,16,REP,David Rogers,17
 Merced,91162,U.S. House,16,DEM,Jim Costa,1
@@ -6257,7 +6257,6 @@ Merced,92182,President,,REP,Ted Cruz,2
 Merced,92182,Proposition 50,,,No,7
 Merced,92182,Proposition 50,,,Yes,27
 Merced,92182,State Assembly,21,DEM,Adam Gray,26
-Merced,92182,State Assembly,21,,Brien J. Rahilly,1
 Merced,92182,State Assembly,21,REP,Greg Opinski,8
 Merced,92182,U.S. House,16,REP,David Rogers,5
 Merced,92182,U.S. House,16,DEM,Jim Costa,28
@@ -6445,6 +6444,7 @@ Merced,92189,President,,REP,Ted Cruz,10
 Merced,92189,Proposition 50,,,No,21
 Merced,92189,Proposition 50,,,Yes,111
 Merced,92189,State Assembly,21,DEM,Adam Gray,95
+Merced,92189,State Assembly,21,,Brien J. Rahilly,1
 Merced,92189,State Assembly,21,REP,Greg Opinski,34
 Merced,92189,U.S. House,16,REP,David Rogers,18
 Merced,92189,U.S. House,16,DEM,Jim Costa,77

